activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+*   `ActiveRecord::Base.transaction` now yields an `ActiveRecord::Transation` object.
```

Comment:
Small spelling mistake on this line: `ActiveRecord::Transation` should instead say `ActiveRecord::Transaction`

Reply:
Thanks, corrected.